test/test_unary_ufuncs.py

```diff
@@ -624,16 +624,6 @@ def test_frexp_assert_raises(self, device):
                 ):
                     torch.frexp(input, out=(mantissa, exponent))
 
-    def test_mvlgamma_argcheck(self, device):
```

Comment:
The docs will need to be updated; currently they say an error will be thrown in this case:

https://pytorch.org/docs/master/special.html